src/parser/priest/discipline/modules/spells/Evangelism.tsx

```diff
@@ -19,19 +18,28 @@ class Evangelism extends Analyzer {
     atonementModule: Atonement,
   };
 
-  _previousEvangelismCast = null;
-  _evangelismStatistics = {};
+  protected atonementModule!: Atonement;
 
-  constructor(...args) {
-    super(...args);
-    this.active = !!this.selectedCombatant.hasTalent(SPELLS.EVANGELISM_TALENT.id);
+  _previousEvangelismCast: CastEvent | null = null;
+  _evangelismStatistics: {
+    [timestamp: number]: {
+      count: number,
+      atonementSeconds: number,
+      healing: number,
+    }
+  } = {};
+
+  constructor(options: any) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.EVANGELISM_TALENT.id);
   }
 
   get evangelismStatistics() {
-    return Object.keys(this._evangelismStatistics).map(key => this._evangelismStatistics[key]);
+    // @ts-ignore
+    return Object.keys(this._evangelismStatistics).map((key: number) => this._evangelismStatistics[key]);
```

Comment:
`return Object.keys(this._evangelismStatistics).map(Number).map((key: number) => this._evangelismStatistics[key]);`

Reply:
Fixed!